website/docs/compaction.md

```diff
@@ -74,22 +67,46 @@ spark-submit --packages org.apache.hudi:hudi-utilities-bundle_2.11:0.6.0 \
 --continous
 ```
 
-### Hudi CLI
-Hudi CLI is yet another way to execute specific compactions asynchronously. Here is an example
+## Executing Async Compaction
+After compactions have been asynchronously scheduled on the Hudi Timeline, you can now execute those compactions with one of the following options
 
-```properties
-hudi:trips->compaction run --tableName <table_name> --parallelism <parallelism> --compactionInstant <InstantTime>
-...
-```
+### Hudi Compactor Utility
+Hudi provides a standalone tool to execute specific compactions asynchronously. Below is an example and you can read more in the [deployment guide](/docs/deployment#compactions)
 
-### Hudi Compactor Script
-Hudi provides a standalone tool to also execute specific compactions asynchronously. Below is an example and you can read more in the [deployment guide](/docs/next/deployment#compactions)
+:::info
+Note: As of version 0.10.0, the `instant-time` parameter is no longer required for the Hudi Compactor Utility. Each spark-submit will execute the earliest scheduled compaction on the Hudi timeline.
+:::
 
 ```properties
 spark-submit --packages org.apache.hudi:hudi-utilities-bundle_2.11:0.6.0 \
 --class org.apache.hudi.utilities.HoodieCompactor \
 --base-path <base_path> \
 --table-name <table_name> \
---instant-time <compaction_instant> \
---schema-file <schema_file>
+--schema-file <schema_file>\
+--instant-time <compaction_instant>
 ```
+
+### Hudi CLI
+Hudi CLI is yet another way to execute specific compactions asynchronously. Here is an example and you can read more in the [deployment guide](/docs/deployment#compactions)
+
+```properties
+hudi:trips->compaction run --tableName <table_name> --parallelism <parallelism> --compactionInstant <InstantTime>
+...
+```
+
```

Comment:
Shall we add a link here to refer to other compaction commands in hudi-cli (https://hudi.apache.org/docs/next/cli#compactions)?  Especially when the user would like to find the compaction instant, `compactions show` commands are helpful.

Reply:
I have it linked just above this